codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/localservice/DafnyPythonLocalServiceProtocolGenerator.java

```diff
@@ -459,7 +459,7 @@ private void generateErrorResponseDeserializerSection(
             writer.write(
               """
               if error.is_Opaque:
-                  return OpaqueError(obj=error.obj)
+                  return OpaqueError(obj=error.obj, alt__text=error.alt__text)
```

Comment:
If you change the native type to `alt_text`, you can change this to
```suggestion
                  return OpaqueError(obj=error.obj, alt_text=error.alt__text)
```

Reply:
This change still needs to go in;
`error` is a Dafny error with `alt__text`,
but `OpaqueError` is the native type with `alt_text`